nixos/modules/system/boot/systemd-nspawn.nix

```diff
@@ -42,8 +42,113 @@ let
     (assertValueOneOf "VirtualEthernet" boolValues)
   ];
 
+  bindMounts = mkOption {
+    type = with types; attrsOf (submodule {
+      options = {
+        source = mkOption {
+          type = str;
+          default = "";
+          description = "Optional mount source path. If omitted, the specified path will be mounted from the host to the same path in the container.";
+        };
+        mountOption = mkOption {
+          type = str;
+          default = "";
+          description = ''
+            Optional mount options. See
+            <link xlink:href="url">https://www.freedesktop.org/software/systemd/man/systemd-nspawn.html#Mount%20Options</link>
```

Comment:
shouldn't the `href` attribute contain a URL

Reply:
could you tell me how to link to 'Mount options' section in the manual?